theme-utils.mjs

```diff
@@ -1014,66 +1014,64 @@ Subscribers:
 }
 
 /*
- Create a Phabricator diff with the given message based on the contents currently in the sandbox.
- Open the phabricator diff in your browser.
- Provide the URL of the phabricator diff.
+ Create a GitHub pull request with the given message based on the contents currently in the sandbox.
+ Open the GitHub pull request in your browser.
+ Provide the URL of the GitHub pull request.
 */
-async function createPhabricatorDiff(commitMessage) {
+async function createGithubPR(commitMessage) {
 
-	console.log('creating Phabricator Diff');
+	console.log('creating GitHub Pull Request');
 
 	let result = await executeOnSandbox(`
 		cd ${sandboxPublicThemesFolder};
 		git branch -D deploy
 		git checkout -b deploy
 		git add --all
 		git commit -m "${commitMessage}"
-		arc diff --create --verbatim
+		gh pr create --fill
```

Comment:
The deploy process fails here with the message: `none of the git remotes configured for this repository point to a known GitHub host.`  when creating the PR.

At this point of the process, the commands are run in the sandbox in the path for the `themes/pub` repo `/home/wpcom/public_html/wp-content/themes/pub`

I tried adding the login command `gh auth login` before creating the PR and while the logging in is successful, the PR creation still fails with the same message.   

<img width="1206" alt="Screenshot 2567-07-26 at 19 17 13" src="https://github.com/user-attachments/assets/310efe32-b415-433c-9db0-2ffcd9c00350">

I'm now trying to figure out from what means that error message means and what's required here. Any thoughts?

Reply:
The remotes look good

<img width="508" alt="Screenshot 2567-07-26 at 19 22 47" src="https://github.com/user-attachments/assets/807b5328-be60-47d1-bfa1-8f45d95988fe">

Reply:
Reading this issue that seems related https://github.com/cli/cli/issues/7269

Reply:
We can add instructions during the process saying "When prompted login in GitHub Enterprise Server typing the A8C GitHub host" 

<img width="293" alt="Screenshot 2567-07-26 at 20 14 46" src="https://github.com/user-attachments/assets/73bda2c9-0bbd-42ec-bc46-026b88b2daf8">

@ehg Maybe you know how to set the $GH_HOST with the A8C Enterprise GitHub host auto-magically in all Dotcom sandboxes.

Reply:
I tried adding a message with instruction but it does't work because during the script process there is no prompt to choose the Enterprise Server.

```
echo "When prompted login in 'GitHub Enterprise Server' typing the A8C GitHub host and using your SSH key."
gh auth login
gh pr create --fill
```
We could pass the host name to `gh auth login --hostname` but I believe that's not an option for safety reasons. See p1721998260238249-slack-C02ECE3ML

The only option I can imagine now is adding the environment variable `export GH_HOST=github.???.com` in your sandboxes and logging in manually before starting the process from local. We can add instructions to the field guide page.

Reply:
> Noting that our $GH_HOST constant is empty and needs to contain the A8C Enterprise GitHub host.

I don't think that helps much, with `gh auth login` it didn't default to the `GH_HOST` when logging in. Also `gh browse` works OK without `GH_HOST`.  I think maybe you could instruct people to read a set up page on the Fieldguide?

Reply:
Resolving, because @miksansegundo has a prompt above that users should already have used `gh auth login`, that worked for me and seems like a good enough solution.